### PR TITLE
data: iamBijoyKar

### PR DESCRIPTION
**Pull request by iamBijoyKar**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots
image

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2947"></a>

### CODE REVIEW

**Review by ChinmayMhatre — DISMISSED** [review]
Looks Good!

**Comment by iamBijoyKar** [comment]
That's great !! Thanks @ChinmayMhatre

File: data/iamBijoyKar.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "links": [
+      {
+        "name": "Twitter",
+        "url": "https://twitter.com/iamBijoyKar",
+        "icon": "FaTwitter"
+      },
+      {
+        "name": "LinkedIn",
+        "url": "https://www.linkedin.com/in/iambijoykar/",
+        "icon": "FaLinkedIn"
```
Comment (kumarsonsoff3) [inline data/iamBijoyKar.json:22]:
```suggestion
        "icon": "FaLinkedin"
```

**Comment by ChinmayMhatre** [comment]
> Please fix your Linkedin icon. It should be `FaLinkedin`. 🛠

Awesome catch! I totally missed it

**Review by eddiejaoude — APPROVED** [review]
Looks good! Great collaboration 👍

**Pull request merged**